package.json

```diff
@@ -26,6 +26,7 @@
     "react-swipeable": "4.3.0",
     "socket.io": "2.1.1",
     "socket.io-client": "2.1.1",
+    "styled-jsx": "3.1.0",
```

Comment:
I think styled-jsx comes bundled with next. At least that's how it used to be. Might have changed though.